### ISSUE TITLE
Color contrast does not meet minimum requirement

### ISSUE THREAD

**Issue by melmai** [melmai/kcalerts-restapi#108]
The tab filter needs to have higher contrast. Increase text color to `#6B747B`

**Result**
Contrast ratio: 4.45:1
Color details
Color contrast requirements vary depending on WCAG conformance level and font size.

**Text color:**
`#6c757d` / `RGB (108, 117, 125)`

**Background color:**
`#f8f9fa` / `RGB (248, 249, 250)`

**Description**
There is insufficient color contrast between the text and background on this page to meet the minimum contrast requirement.

Text and background colors that conflict or have too little contrast between each other can cause readability issues for users who are color blind or have low vision (vision loss equivalent to approx. 20/40 vision).

To meet the WCAG contrast requirement, you should have a contrast ratio of 4.5:1 for normal text, and 3:1 for large text (18pt and above, or 14pt bold).

SiteImprove: [https://my2.siteimprove.com/Accessibility/115268/59385125470/NextGen/Issue/1?ruleId=69&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=22&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1&siteTarget.issueKinds=1,2]
